tests/test_pay.py

```diff
@@ -3586,14 +3586,15 @@ def test_keysend(node_factory):
 def test_keysend_strip_tlvs(node_factory):
     """Use the extratlvs option to deliver a message with sphinx' TLV type, which keysend strips.
     """
-    amt = 10000
+    amt = 10**7
```

Comment:
This amount change is only here so we get above the 1% reserve to later be able to send back from `l1` to `l2`.